src/Service/HaalCentraalToStufBGService.php

```diff
@@ -132,13 +133,43 @@ public function getAllRelatives(Source $source, array $ingeschrevenPersoon): ?ar
                     'burgerservicenummer' => implode(',', $bsns),
                 ];
 
-                if (isset($ingeschrevenPersoon['verblijfplaats']['postcode']) === true && $ingeschrevenPersoon['verblijfplaats']['huisnummer']) {
-                    $query['verblijfplaats__postcode']   = $ingeschrevenPersoon['verblijfplaats']['postcode'];
-                    $query['verblijfplaats__huisnummer'] = $ingeschrevenPersoon['verblijfplaats']['huisnummer'];
+                $people = $this->fetchPerson($source, '', $query);
+
+                if ($people === null) {
+                    continue;
                 }
 
-                $people               = $this->fetchPerson($source, '', $query,);
-                $fetchedPeople[$type] = $people['_embedded']['ingeschrevenpersonen'];
+                $foundPeople          = new ArrayCollection($people['_embedded']['ingeschrevenpersonen']);
+                $fetchedPeople[$type] = $foundPeople->filter(
+                    function ($person) use ($ingeschrevenPersoon) {
+                        $comp = true;
+                        if (isset($ingeschrevenPersoon['verblijfplaats']['postcode']) === true && isset($person['verblijfplaats']['postcode']) === true) {
+                            $comp = $comp && $ingeschrevenPersoon['verblijfplaats']['postcode'] === $person['verblijfplaats']['postcode'];
```

Comment:
Waarom overal hier checken op "= $comp &&" als je $comp op 145 sws standaard op true zet?

Reply:
Dit is een uniformiteitsdingetje, hij kan hier dus false worden etc. Hierdoor maakt het dus niet uit als we ooit de volgorde van deze checks omdraaien ;)

Reply:
Oke prima